src/json_request_translator.cc

```diff
@@ -146,12 +146,14 @@ JsonRequestTranslator::JsonRequestTranslator(
     stream_translator_.reset(new RequestStreamTranslator(
         *type_resolver, output_delimiters, std::move(request_info)));
     writer = stream_translator_.get();
+    writer->set_use_strict_base64_decoding(false);
```

Comment:
I believe you only need to set this to one object:

proto_writer_

in this line
https://github.com/grpc-ecosystem/grpc-httpjson-transcoding/blob/master/src/request_message_translator.cc#L54

Reply:
Thank you for the feedback! I'll try it and come back to you.

Reply:
@qiwzhang Yes, setting that one object fixes the issue. I updated the branch to reflect it.